containers/Dockerfile

```diff
@@ -21,7 +21,7 @@ COPY --from=azure-cli /usr/local/bin/az /usr/bin/az
 RUN yum install epel-release -y && \
     yum install -y git python39 python3-pip jq gettext && \
     python3.9 -m pip install -U pip && \
-    git clone https://github.com/redhat-chaos/krkn.git --branch v1.1.1 /root/kraken && \
+    git clone https://github.com/redhat-chaos/krkn.git --branch v1.2.0 /root/kraken && \
```

Comment:
is there a v1.2.0 tag already on krkn? I'm not seeing

Reply:
Hey @paigerube14, we will be drafting a release immediately after this PR gets merged. Release draft can be found here: https://github.com/redhat-chaos/krkn/releases/tag/untagged-9c77ab3dfbd0919860d4.